### PR TITLE
related collections

### PR DESCRIPTION
**Pull request by alansutherland**

### CODE REVIEW

File: components/search/SearchRelatedCollections.stories.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    template: `
+      <b-container
+        class="mt-3"
+      >
+        <RelatedCollections
+          :title="$t('collectionsYouMightLike')"
+          :related-collections="relatedCollections"
+        />
+      </b-container>`
+  }));
```
Comment (mirjamverloop) [inline components/search/SearchRelatedCollections.stories.js:60]: The styling of the chips on search pages has changed slightly when I compare it with production. Has that been done on purpose? To keep things consistent?

File: components/generic/RelatedCollections.vue [hunk 2]
```diff
@@ -4,21 +4,22 @@
     data-qa="related collections"
```
Comment (mirjamverloop) [inline components/generic/RelatedCollections.vue:4]: This div (or its content) is not aligned is not aligned (left) with the title and description above. Or has this been fixed as part of the other PR?

File: components/generic/RelatedCollections.vue [hunk 3]
```diff
@@ -29,50 +30,59 @@
     },
 
     props: {
-      query: {
+      title: {
```
Comment (mirjamverloop) [inline components/generic/RelatedCollections.vue:33]: Is this prop still necessary if the title is "Collections you might like" everywhere?

File: components/generic/RelatedCollections.vue [hunk 4]
```diff
@@ -29,50 +30,59 @@
...
             pathMatch: getEntitySlug(id, name)
           }
         });
+      },
+      collectionTitle(item) {
+        if (item.prefLabel) {
+          return item.prefLabel[this.$i18n.locale];
+        } else {
+          return item.name;
+        }
```
Comment (mirjamverloop) [inline components/generic/RelatedCollections.vue:75]: Perhaps
`return item.prefLabel ? item.prefLabel[this.$i18n.locale] : item.name`

File: components/generic/RelatedCollections.vue (hunk 2 above)
Comment (mirjamverloop) [reply components/generic/RelatedCollections.vue:4]: On collection pages, like http://localhost:3000/en/collections/topic/128-migration

File: components/generic/RelatedCollections.vue [hunk 5]
```diff
@@ -29,50 +30,59 @@
...
+      },
+      imageUrl(item) {
+        if (typeof item.image === 'undefined' && typeof item.isShownBy === 'undefined') {
+          return '';
+        }
+        if (item.image) {
+          return item.image + '&size=w200';
+        } else if (item.isShownBy.thumbnail) {
+          return item.isShownBy.thumbnail + '&size=w200';
+        }
```
Comment (mirjamverloop) [inline components/generic/RelatedCollections.vue:85]: The images are now squeezed, they are tiny, so hard to see. But perhaps adding `object-fit: cover;` on the badge img would improve the way the images are displayed.

File: components/generic/RelatedCollections.stories.js [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+    i18n,
+    store,
+    data() {
+      return {
+        relatedCollections: [
+          {
+            name: "Emilio Pucci",
+            identifier: "http://data.europeana.eu/agent/base/69103",
+            image: "https://api.europeana.eu/api/v2/thumbnail-by-url.json?uri=http%3A%2F%2Fwww.museum-digital.de%2Fdata%2Fberlin%2Fimages%2Fimport_17%2F201601%2F29193203582.jpg&type=IMAGE"
+          },
```
[the remark this answers is not in the document]
Comment (alansutherland) [reply components/generic/RelatedCollections.stories.js:36]: fixed

File: components/search/SearchRelatedCollections.stories.js (hunk 1 above)
Comment (alansutherland) [reply components/search/SearchRelatedCollections.stories.js:60]: yes the border has been added in place of the box-shadow following the latest design

File: components/generic/RelatedCollections.vue (hunk 3 above)
Comment (alansutherland) [reply components/generic/RelatedCollections.vue:33]: I can remove it but it was added when we had multiple titles and if in future multiple titles are requested again then they can be added easily.

File: components/generic/RelatedCollections.vue (hunk 4 above)
Comment (alansutherland) [reply components/generic/RelatedCollections.vue:75]: yes thats better

File: components/generic/RelatedCollections.vue (hunk 2 above)
Comment (alansutherland) [reply components/generic/RelatedCollections.vue:4]: ok removed the padding

File: components/search/SearchRelatedCollections.stories.js (hunk 1 above)
Comment (alansutherland) [reply components/search/SearchRelatedCollections.stories.js:60]: I changed the hover background colour to match the border colour but I have now changed it back to #f5f5f5 as it was before

File: components/generic/RelatedCollections.vue (hunk 5 above)
Comment (alansutherland) [reply components/generic/RelatedCollections.vue:79]: thanks I hadn't noticed since they're so tiny

**Pull request merged**